cosign.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: cosign
+  version: 1.13.1
+  description: Container Signing
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    repositories:
+      - https://packages.wolfi.dev/os
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/sigstore/cosign/archive/v${{package.version}}/cosign-v${{package.version}}.tar.gz
+      expected-sha256: 496583e9a3e07f16ff18f3734ca69398ef7091336fe74adaf8012715656bb920
+  - runs: |
+      CGO_ENABLED=0 go build \
+        -o "${{targets.destdir}}/usr/bin/cosign" ./

```

Comment:
probably needs to be `go build ./...`.

Reply:
I think this should be `go build ./cmd/cosign`

Reply:
Can we also inject the version in the ldflags?

Reply:
i injected the version but git state is not there in the release tar archive so I just leave it out.